app/src/main/java/org/fossasia/badgemagic/adapter/SaveAdapter.kt

```diff
@@ -13,9 +13,11 @@ import org.fossasia.badgemagic.R
 import org.fossasia.badgemagic.data.fragments.BadgeConfig
 import org.fossasia.badgemagic.data.fragments.ConfigInfo
 import org.fossasia.badgemagic.util.MoshiUtils
+import org.fossasia.badgemagic.util.Resource
 
 class SaveAdapter(private val context: Context?, private val list: List<ConfigInfo>, private val listener: OnSavedItemSelected) : RecyclerView.Adapter<SaveAdapter.SaveItemHolder>() {
     private var selectedPosition: Int = -1
+    private val resourceHelper = Resource()
```

Comment:
Please only use in ViewModels

Reply:
what about this -> https://github.com/fossasia/badge-magic-android/blob/development/app/src/main/java/org/fossasia/badgemagic/adapter/DrawableItemHolder.kt

Reply:
Use view.context there

Reply:
Wow, that is so cool. Thanks sir